### ISSUE TITLE
GDAX triggering buy signal not actually buying anything on btc eur

### ISSUE THREAD

**Issue by jellebens** [DeviaVir/zenbot#834]
GDAX triggering buy signal not actually buying anything on btc eur

i can do it manually though

**Comment by Kattjakt** [comment]
Try running `extensions/exchanges/gdax/update-products.sh`!

**Comment by mahdihrz** [comment]
I have got the same issue and now it is working after running _extensions/exchanges/gdax/update-products.sh_

**Comment by benjamincburns** [comment]
Updating products will help if you're running an older version, but it won't fix all of the problems. See [discussion](https://github.com/DeviaVir/zenbot/issues/868#issuecomment-354963455) over on #868.

**Comment by davestephens** [comment]
#868 is not not the same issue. There seems to be an element of unreliability since switching to the gdax websocket API rather than the REST API. My assumption is that Zenbot loses the connection but doesn't spot it, "sends" a buy or sell which never actually make it to the exchange. It then doesn't do any more buy or sells because it doesn't get a response to the buy or sell it never actually did. The status constantly saya "buying" or "selling" until you restart it.

**Comment by DeviaVir** [comment]
Do you still think this issue is around today in the latest `unstable`?

**Comment by abelardojarab** [comment]
yes it is and it is causing big problem with market changing so fast. the best sells at a given price and btc value goes down (ok good until now) but then the value of btc bounces back and the bot sends 'buy' signal but it NEVER BUYS again, so you end with less BTC than when you started.

I see that you are closing all tickets in favor of 868; but this is not the same thing.

please dont ignore this issue it i causing big losses... thanks for your great work btw
